### DISCUSSION TITLE
Add argument in pkg_manual() to allow exclusion of the package overview page

### DISCUSSION THREAD

**Discussion by TimTaylor** [yihui/litedown#51] [Ideas]
It would be useful to be able to optionally exclude the `{pkgname}-package.Rd` overview from the generated manual. Perhaps `pkg_manual()` could gain an `overview` argument to control this?

**Comment by yihui** [comment]
Sure: a26e5d8892fe3ba76a2e1a6285aaf9bc6d59b23d
